extension/encoding/awslogsencodingextension/README.md

```diff
@@ -262,39 +262,50 @@ If you're using the old format values you should update the encoding extension c
+| `eventName`                                               | `rpc.method`                                              |
+| `eventSource`                                             | `rpc.service`                                             |
+| `eventType`                                               | `rpc.system`                                              |
+| `tlsDetails.clientProvidedHostHeader`                     | `server.address`                                          |
+| `sourceIPAddress`                                         | `source.address`                                          |
+| `tlsDetails.cipherSuite`                                  | `tls.cipher`                                              |
+| `tlsDetails.tlsVersion`                                   | `tls.protocol.version`                                    |
+| `userAgent`                                               | `user_agent.original`                                     |
+| `userIdentity.type`                                       | `aws.principal.type`                                      |
+| `userIdentity.principalId`                                | `aws.principal.id`                                        |
+| `userIdentity.arn`                                        | `aws.principal.arn`                                       |
+| `userIdentity.accountId`                                  | `aws.owner.account_id`                                    |
+| `userIdentity.accessKeyId`                                | `aws.access_key.id`                                       |
+| `userIdentity.userId`                                     | `user.id`                                                 |
+| `userIdentity.userName`                                   | `user.name`                                               |
+| `userIdentity.identityStoreArn`                           | `aws.identity_store.arn`                                  |
+| `userIdentity.invokedBy`                                  | `aws.invoked_by`                                          |
+| `userIdentity.sessionContext.attributes.creationDate`     | `aws.session_context.attributes.creation_date`            |
+| `userIdentity.sessionContext.attributes.mfaAuthenticated` | `aws.session_context.attributes.mfa_authenticated`        |
+| `userIdentity.sessionContext.sessionIssuer.type`          | `aws.session_context.issuer.type`                         |
+| `userIdentity.sessionContext.sessionIssuer.principalId`   | `aws.session_context.issuer.principal_id`                 |
+| `userIdentity.sessionContext.sessionIssuer.arn`           | `aws.session_context.issuer.arn`                          |
+| `userIdentity.sessionContext.sessionIssuer.accountId`     | `aws.session_context.issuer.account_id`                   |
+| `userIdentity.sessionContext.sessionIssuer.userName`      | `aws.session_context.issuer.user_name`                    |
```

Comment:
Other than `user.id` and `user.name`, I'm wondering if these should share a common prefix. Is there precedent for these attribute names?

Reply:
Good suggestion and it makes sense. I think `user_identity` is suitable and I will add this to new fields I am introducing through this PR. For the rest, we will have to loop back and introduce it as a breaking change.